kmod/src/lock.c

```diff
@@ -1415,24 +1430,25 @@ static void lock_shrink_worker(struct work_struct *work)
  * mode which will prevent the lock from being freed when the null
  * response arrives.
  */
-static int scoutfs_lock_shrink(struct shrinker *shrink,
-			       struct shrink_control *sc)
+static unsigned long lock_scan_objects(struct shrinker *shrink,
```

Comment:
This shrinker patch should be merged with the previous one so that we have one shrinker commit where we can review all the changes and see consistency, etc.